images/virt-launcher-lib-builder/werf.inc.yaml

```diff
@@ -0,0 +1,51 @@
+---
+image: {{ $.ImageName }}
+fromImage: base-alt-p10
+shell:
+  install:
```

Comment:
It should be 2 stages: beforeInstall with `apt-get update && apt-get install --yes ...` and install with compiliing.
Last steps `# Remove build packages and sources.` are not required.